docs/getting-started/custom-finders.mdx

```diff
@@ -1,13 +1,8 @@
 ---
-title: Getting started
+title: Getting started - custom finders
```

Comment:
"Getting started" is redundant here. Maybe "Custom finders setup" or just "Custom finders"?

Reply:
And title for `getting-started/native` is only "Setup". It could be named in the same convention.